lib/node_modules/@stdlib/math/base/special/bessely0/benchmark/python/benchmark.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
```

Comment:
For SciPy benchmarks, we typically name the benchmark file `benchmark.scipy.py`. This allows us to, if need be, have multiple benchmarks for different Python libraries. If the function is a standard library function, we name just `benchmark.py`. (Aside: I have already updated `besselj0` to this effect.)